docs/pages/_app.js

```diff
@@ -178,7 +178,6 @@ function AppWrapper(props) {
         wordmarkSvg: muiSvgWordmarkString,
         versions: [
           { text: `v${materialPkgJson.version}`, current: true },
-          { text: `v5`, href: `https://mui.com${languagePrefix}/material-ui/getting-started/` },
```

Comment:
This should be redirected to `v5.mui.com` instead of removed, right?

Reply:
Done!